Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/common/use-hash-url-params.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { createContext, useCallback, useContext, useMemo } from 'react';
+import { useLocation } from 'react-router-dom';
+
+import { IHaveChildrenProps } from '../../components/common/Props';
+
+interface IHashUrlParamsContext {
+    state: { params: unknown };
+    actions: {
+        changeHash: (hashParameter: number) => void;
+    };
+}
+
+type IHashUrlParamProviderProps = IHaveChildrenProps
+
+const HashUrlParamContext = createContext<IHashUrlParamsContext>({} as IHashUrlParamsContext);
+
+const HashUrlParamProvider = ({ children }: IHashUrlParamProviderProps) => {
+    const location = useLocation();
+    const params = useMemo(() => {
+        const { hash } = location;
+
+        return hash;
+    }, [ location ]);
+
+    const changeHash = useCallback(
+        (param: number) => {
```

Comment:
I used number, because **orderBy** parameter in **use-problems.tsx** was ### number. I changed the parameter in the useCallBack to be **number** and converted **orderBy** to be **.toString()**.